tests/perf/auto_perf_sheriffing/conftest.py

```diff
@@ -64,29 +64,44 @@ def record_unsuited_for_backfill(test_perf_alert, request):
         )
 
 
-@pytest.fixture
-def linux_signature(test_repository, test_perf_framework) -> PerformanceSignature:
-    linux_platform = MachinePlatform.objects.create(
-        os_name='linux', platform='linux', architecture='x86'
+@pytest.fixture(params=['windows', 'linux'])
+def platform_specific_signature(
+    test_repository, test_perf_framework, request
+) -> PerformanceSignature:
+    new_platform = MachinePlatform.objects.create(
+        os_name=request.param, platform=request.param, architecture='x86'
     )
-    return create_perf_signature(test_perf_framework, test_repository, linux_platform)
+    return create_perf_signature(test_perf_framework, test_repository, new_platform)
 
 
 @pytest.fixture
-def linux_perf_alert(linux_signature, test_perf_alert_summary) -> PerformanceAlert:
-    return create_perf_alert(summary=test_perf_alert_summary, series_signature=linux_signature)
+def platform_specific_perf_alert(
+    platform_specific_signature, test_perf_alert_summary
+) -> PerformanceAlert:
+    perf_alert = create_perf_alert(
+        summary=test_perf_alert_summary, series_signature=platform_specific_signature
+    )
+
+    # provide platform hints for client tests
+    perf_alert.platform = platform_specific_signature.platform.platform
+
+    return perf_alert
 
 
 @pytest.fixture
-def record_ready_for_processing(linux_perf_alert, record_context_sample):
-    report = BackfillReport.objects.create(summary=linux_perf_alert.summary)
+def record_ready_for_processing(platform_specific_perf_alert, record_context_sample):
+    report = BackfillReport.objects.create(summary=platform_specific_perf_alert.summary)
     record = BackfillRecord.objects.create(
-        alert=linux_perf_alert,
+        alert=platform_specific_perf_alert,
         report=report,
         status=BackfillRecord.READY_FOR_PROCESSING,
     )
     record.set_context(record_context_sample)
     record.save()
+
+    # provide platform hints for client tests
+    record.specific_to_platform = platform_specific_perf_alert.platform
```

Comment:
Don't we have the value in `record.alert.platform`?

Reply:
I've updated the related code so it's not that brittle.